app/src/main/java/com/likeminds/feedsx/posttypes/view/adapter/databinder/ItemPostSingleVideoViewDataBinder.kt

```diff
@@ -64,16 +47,41 @@ class ItemPostSingleVideoViewDataBinder constructor(
             listener
         )
 
-        //TODO: Migrate to exo player
-        val video: Uri =
-            Uri.parse(data.attachments.first().attachmentMeta.url)
+        if (data.fromPostLiked || data.fromPostSaved) {
+            return
```

Comment:
create a callback function to update the values of `fromPostLiked` and `fromPostSaved`